packages/desktop-client/src/components/schedules/EditSchedule.js

```diff
@@ -432,10 +433,15 @@ export default function ScheduleDetails() {
           <FormLabel title="Payee" />
           <PayeeAutocomplete
             value={state.fields.payee}
-            inputProps={{ placeholder: '(none)' }}
+            inputProps={{
+              onBlur: () => setPayeeFieldFocused(false),
+              onFocus: () => setPayeeFieldFocused(true),
+              placeholder: '(none)'
+            }}
```

Comment:
suggestion: would it be possible to move the fix to `PayeeAutocomplete` component? Seems quite redundant to do it 2x: once here and once in `TransactionsTable`. Ideally we do it once and it fixes it in all the places where the component is used.